src/commands/close.js

```diff
@@ -0,0 +1,23 @@
+const getConfig = require('../config/get-config.js');
+const config = getConfig();
+module.exports = {
+  prefix: 'close',
+  description: 'Closes the channel.',
+  command: async function (message) {
+    const target = message.channel;
+    //check for log channel
+    const log = message.guild.channels.cache.find(
+      (channel) => channel.name === config.LOG_MSG_CHANNEL
+    );
+    //check for user permissions
+    if (!message.member.hasPermission('MANAGE_CHANNELS')) {
+      console.log('Missing permissions.');
+    }
+    if (!log) {
+      console.log('Log channel not found.');
+      return;
+    }
+    target.delete().catch((e) => console.error(e));
+    log.send(`${message.author} deleted a channel.`);
```

Comment:
Do we want this to fail silently? If we are adding `.catch` to things that can fail we can always add a try/catch around the entire command function instead, so we still log the error, but don't send a message back saying the channel was deleted when it wasn't.